servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/TerminalSignalConsumer.java

```diff
@@ -40,4 +42,14 @@ default void onError(Throwable throwable) {
      */
     default void onCancel() {
     }
+
+    /**
+     * Creates a new {@link TerminalSignalConsumer} from {@link Runnable}.
+     *
+     * @param onFinally a {@link Runnable} to run on any terminal signal
```

Comment:
```suggestion
     * @param onFinally a {@link Runnable} to adapt to the returned {@link TerminalSignalConsumer} such that {@link Runnable#run()} is invoked for each call to {@link TerminalSignalConsumer#onError()}, {@link TerminalSignalConsumer#onComplete()} and {@link TerminalSignalConsumer#onCancel()}
```

Reply:
This should be `{@link {@link TerminalSignalConsumer#onError(Throwable)}}`